source/index.html

```diff
@@ -1,12 +1,188 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Мишка</title>
+    <title>Мишка - милые штуки ручной работы для дома</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="main-header">
+      <h1 class="main-header__title">Милые штуки ручной работы для дома</h1>
+      <a class="main-header__logo" href="#">
+        <img class="main-header__logo-image" src="#" width="86" height="35" alt="Логотип магазина товаров ручной работы «Мишка»">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item">
+              <a href="#">Каталог товаров</a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">Вязание на заказ</a>
+            </li>
+          </ul>
+          <ul class="main-nav__list user-list">
+            <li class="user-list__item">
+              <a href="#"><span class="visually-hidden">Поиск</span></a>
+            </li>
+            <li class="user-list__item">
+              <a href="#">Корзина: <span>пока пуста</span></a>
+            </li>
+          </ul>
+          <ul class="main-nav__list sections-list">
+            <li class="sections-list__item sections-list__item--interior">
+              <a href="#">Предметы интерьера</a>
+            </li>
+            <li class="sections-list__item sections-list__item--toys">
+              <a href="#">Детские игрушки</a>
+            </li>
+          </ul>
+          <div class="main-nav__delivery"><a href="#">Бесплатная доставка по Росси</a></div>
+        </div>
+
+      </nav>
+    </header>
+
+    <main class="main-container">
+      <h1 class="main-container__title">Вязаные корзинки</h1>
+
+      <section class="featured-product">
+        <div class="featured-product__wrapper">
+          <h2 class="featured-product__title">Товар недели</h2>
+          <p class="featured-product__description">Экологически чистая пряжа,
+            ручная работа. Доступны в шести расцветках и трёх размерах.</p>
+          <dl class="featured-product__list">
+            <dt class="featured-product__item">Цвет:</dt>
+            <dd class="featured-product__item-description">тиффани/серый</dd>
+            <dt class="featured-product__item">Диаметр:</dt>
+            <dd class="featured-product__item-description">25 см</dd>
+            <dt class="featured-product__item">Высота:</dt>
+            <dd class="featured-product__item-description">15 см</dd>
+          </dl>
+          <img class="featured-product__image"src="" alt="Фото товара недели">
```

Comment:
```suggestion
          <img class="featured-product__image" src="" alt="Фото товара недели">
```
1. Отсутствует пробел между атрибутами.
2. Нет пути к изображению в `src`.